### PR TITLE
Release v2.0.0

### PR DESCRIPTION
**Pull request by rschouten97**

### CODE REVIEW

**Comment by rschouten97** [comment]
The actionMessage variables should be generic. Including specific details like "with AccountReference: $($actionContext.References.Account | ConvertTo-Json)" can be omitted since the error message will provide this information.

**Comment by rschouten97** [comment]
The delete action logs old and new values, which aren't available at that time. This can be removed.

File: fieldMapping.correlateOnly.json
Comment (Rick-Jongbloed) [inline fieldMapping.correlateOnly.json:1]: This file can be removed i think, as there's also a mapping file in the correlateOnly directory

Comment (rschouten97) [reply fieldMapping.correlateOnly.json:1]: Has been removed.

**Review by rhouthuijzen — DISMISSED** [review]
As discussed the connector is successfully implemented two times.

**Review by rhouthuijzen — APPROVED** [review]
As discussed the connector is successfully implemented two times.

**Pull request merged**